Resources/Prototypes/Reagents/medicine.yml

```
@@ -316,6 +316,13 @@
   metabolisms:
     Medicine:
       effects:
+      - !type:AdjustReagent
+        conditions:
+          - !type:ReagentThreshold
+            reagent: Histamine
+            min: 45
```

```suggestion
        conditions:
        - !type:ReagentThreshold
          reagent: Histamine
          min: 45
```
one more :trollface: 
see the [yaml conventions](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)

---

Woe, one day blinding stew upon me, 3 pairs of eyes did not see the indentation issues. Fixed